File: 2137-final-value-of-variable-after-performing-operations/2137-final-value-of-variable-after-performing-operations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int ans = 0;
        for (String operation : operations) {
            if (operation.equals("--X") || operation.equals("X--")) {
                ans--;
            } else if (operation.equals("++X") || operation.equals("X++")) {
                ans++;
            }
        }
        return ans;
    }
}

File: 2137-final-value-of-variable-after-performing-operations/2137-final-value-of-variable-after-performing-operations.js
var finalValueAfterOperations = function(operations) {
    let ans = 0;
    for (let operation of operations) {
        if (operation === "--X" || operation === "X--") {
            ans--;
        } else if (operation === "++X" || operation === "X++") {
            ans++;
        }
    }
    return ans;
};